examples/AcousticWave2D_codegen.py

```diff
@@ -78,14 +81,15 @@ def damp_boundary(damp):
         self.dm = dm
 
     def Forward(self):
-        fw = ForwardOperator(self.m, self.src, self.damp, self.rec, self.u, time_order=self.t_order, spc_order=self.s_order)
+        fw = ForwardOperator(self.m, self.src, self.damp, self.rec, self.u, time_order=self.t_order, profile=True,
+                             spc_order=self.s_order, auto_tune=self.auto_tune, cache_blocking=self.cache_blocking)
         fw.apply()
-        return (self.rec.data, self.u.data)
+        return self.rec.data, self.u.data
```

Comment:
removes redundant parentheses. It actually doesn't change anything but good code should not have stuff that is not needed.